app/scripts/directives/serviceInstanceIntegrations.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+angular.module('openshiftConsole').component('serviceInstanceIntegrations', {
```

Comment:
for my own knowledge can you explain what this is for? What does it do

Reply:
This is modelled from the `serviceInstanceBindings` directive, it takes an array of integrations and renders each one as a single `serviceInstanceIntegration` component.